src/globalStyles/global.scss

```diff
@@ -15,4 +15,12 @@ th.reactable-header-sort-asc:after {
   content: '\25BC';
   padding-left: 10px;
   padding-bottom: 5px;
-}
\ No newline at end of file
+}
+
+/* makes react-bootstrap-table work in flexbox layout */
+.react-bs-table-container,.react-bs-table,.react-bs-container-body {
```

Comment:
Putting aside whether we use react-bs-table or not, this would be better applied to the component instance itself.